inginious/agent/docker_agent/__init__.py

```diff
@@ -441,15 +440,27 @@ async def create_student_container(self, parent_info, socket_id, environment_nam
                 return
 
             environment = self._containers[environment_type][environment_name]["id"]
-            runtime = self._containers[environment_type][environment_name]["runtime"]
+
+            # If the user did not enter any environment name as input, it is "default"
+            if environment_name != "default":
```

Comment:
I'm quite confused about this.

Reply:
Well, because these 2 lines have no sense :) 
Removed.